platform/admin/docs/src/main/resources/_contents/managing-contents.adoc

```diff
@@ -246,7 +243,7 @@ To do so, follow these instructions:
 
 . Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
```

Comment:
These images are broken for me.

![screen shot 2016-03-01 at 1 56 49 pm](https://cloud.githubusercontent.com/assets/11355332/13441640/8af3b40a-dfb5-11e5-95a5-2ab70b2029cd.png)